cmd/list.go

```diff
@@ -35,18 +35,34 @@ var listCmd = &cobra.Command{
 	RunE:  runList,
 }
 
-func runList(cmd *cobra.Command, args []string) error {
+func roleList(all bool) (string, error) {
 	client, err := creds.GetClient(region)
 	if err != nil {
-		return err
+		return "", err
 	}
-	roles, err := client.Roles(showAll)
+	roles, err := client.Roles(all)
```

Comment:
Maybe we should just show all roles by default? The difference between "console" and "app" roles could be confusing to users

Reply:
Yeah, that does seem to get folks. I'll make that change.